src/components/BigNumberPad.js

```diff
@@ -0,0 +1,72 @@
+import React, {PureComponent} from 'react';
+import {
+    Text, TouchableOpacity, View,
+} from 'react-native';
+import PropTypes from 'prop-types';
+import styles from '../styles/styles';
+
+const propTypes = {
+    numberPressed: PropTypes.func.isRequired,
+};
+
+const padNumbers = [
+    ['1', '2', '3'],
+    ['4', '5', '6'],
+    ['7', '8', '9'],
+    ['.', '0', '<'],
+];
+
+class BigNumberPad extends PureComponent {
```

Comment:
I'm not too knowledgeable here, but just want to confirm the usage of PureComponent instead of React.Component. I believe it's fine in this case because all child components depend on the BigNumberPad to update and re-render, so we don't need to worry about the children not updating. Is that correct?

Reply:
As `BigNumberPad` component has a single function callback property, I thought it would be better to use `PureComponent`. 
> I believe it's fine in this case because all child components depend on the BigNumberPad to update and re-render, so we don't need to worry about the children not updating. Is that correct?

Yes, if I understood it correctly.